File: 02-JavaScript/1-Javascript/2-Exercices/ExPart2(Objets)/Ex3-POO/module/js/personne.js
export default class personne {
    constructor(param1, param2, param3, param4, param5, param6) {
        this.Genre = param1;
        this.Nom = param2;
        this.Prenom = param3;
        this.Naissance = param4;
        this.Telephone = param5;
        this.Email = param6;
    }
    Display = () => {
        return `
        <tr>
        <td>${this.Genre}</td>
        <td>${this.Nom}</td>
        <td>${this.Prenom}</td>
        <td>${this.Naissance}</td>
        <td>${this.Telephone}</td>
        <td>${this.Email}</td>  
        </tr> 
        `
    }
}

File: 09-AspNETCore/FormationAspNETCore/Cours/CoursAPI/CoursAPI/Models/Personne.cs
namespace CoursAPI.Models
{
    public class Personne
    {
        private int id;
        private string nom;
        private string prenom;
        private string email;
        private string telephone;

        public Personne()
        {

        }

        public string Nom { get => nom; set => nom = value; }
        public string Prenom { get => prenom; set => prenom = value; }
        public string Email { get => email; set => email = value; }
        public string Telephone { get => telephone; set => telephone = value; }
        public int Id { get => id; set => id = value; }
    }
}
